src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -10,13 +10,15 @@
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <DebugSymbols>true</DebugSymbols>
     <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+     <!-- Removing SymbolPackageFormat snupkg since it only supports *.pdb and some more
```

Comment:
Don't remove this - we should just not be including the README in the snupkg. We still want to produce the snupkg though.

Reply:
from my understanding it still does produce the package but if you mention it, it filters out files with certain extensions only for packaging and md files are filtered so the pr validation is failing since it's not packaged.

Reply:
That's a long thread and I'm not really sure the context here...what do you mean the PR validation is failing? Do you have a link to a build with the failure?

If we can't embed the README then it's reasonable to just move this stuff into a nuspec file I think and hardcode the description there.

Reply:
The thread mentions there's a fix for it out - we should also be able to update the SDK version we're using as well. As long as we still targeting netstandard2.1 I believe that should be fine.

Reply:
This is the failing build: https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=141092&view=logs&j=ccb1d05f-c091-5851-4ae8-18f7fca67013&t=0f04aa1d-081c-58aa-9042-59cde02cc630

Let me check, I've only read till the part that said to remove the snupkg.